README.md

```diff
@@ -1,2 +1,141 @@
-# did-key-driver
-A did-io driver for the DID "key" method
+# did:key method driver _(did-key-method)_
```

Comment:
Repo name is `did-key-driver`, package.json says `did-io-key`, and here it says `did-key-method`.  They should match.  Related is maybe changing any of these drivers to use a form like "did-method-TYPE", similar to how other plugin systems work.

Reply:
Oh right, thanks!

Reply:
@davidlehn Changed the README and package.json to use `did-method-key` to match other drivers.
Can you or @dlongley rename this repo to `did-method-key` as well? (I don't seem to have admin)

Reply:
Renamed to `did-method-key`